app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import dayjs from "dayjs";
+import { MenuVertical, Clock } from "neetoicons";
+import { Typography, Dropdown, Tooltip, Avatar, Tag } from "neetoui";
+
+const { Menu, MenuItem } = Dropdown;
+
+const Note = ({ note }) => (
+  <div className="mb-3 w-full border border-gray-300 bg-white p-4 shadow-md dark:border-gray-700 dark:bg-gray-800 ">
+    <div className="flex justify-between">
+      <Typography style="h4" weight="bold">
+        {note.title}
+      </Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <Menu>
+          <MenuItem.Button>Edit</MenuItem.Button>
+          <MenuItem.Button>Delete</MenuItem.Button>
```

Comment:
Buttons that perform irreversible actions like `Deleting` or `Stopping Subscription` etc should have the `style` as `danger`.

Refer: https://academy.bigbinary.com/learn-react/pages/semantic-html-and-accessibility